packages/common/test/content/manageSchedule.test.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  deleteSchedule,
+  getSchedule,
+  maybeUpdateSchedule,
+  setSchedule,
+} from "../../src/content/manageSchedule";
+import {
+  IHubSchedule,
+  IHubScheduleResponse,
+} from "../../src/core/types/IHubSchedule";
+import { MOCK_HUB_REQOPTS } from "../mocks/mock-auth";
+import { IHubEditableContent } from "../../src/core/types/IHubEditableContent";
+import * as fetchMock from "fetch-mock";
+
+describe("manageSchedule", () => {
+  afterEach(() => {
+    fetchMock.restore();
+  });
+
+  it("getSchedule: returns null if no schedule is set", async () => {
```

Comment:
We should probably update the test names since we no longer return null. Same goes for the other tests in this file.